src/styles.css

```diff
@@ -37,6 +37,13 @@ header {
     background-position: center center;
 }
 
+.resized{
```

Comment:
Have you considered using a preprocessor, like scss, for your css? Would introduce some more structure.